.eslintrc.js

```diff
@@ -15,7 +18,38 @@ const config = {
                 noStrings: true
             }
         ]
-    }
+    },
+    overrides: [
+        {
+            files: ['**/*.gql.js'],
+            processor: '@graphql-eslint/graphql'
+        },
+        {
+            files: ['*.graphql'],
+            parser: '@graphql-eslint/eslint-plugin',
+            parserOptions: {
+                operations: graphQLConfig.documents,
+                schema: graphQLConfig.schema,
+                schemaOptions: {
+                    assumeValid: true,
+                    method: 'GET'
+                },
+                skipGraphQLConfig: true
+            },
+            plugins: ['@graphql-eslint'],
+            rules: {
+                '@graphql-eslint/known-directives': 'error',
```

Comment:
I've only added a small subset of rules here from `packages/graphql-cli-validate-magento-pwa-queries/lib/index.js`. When I added all the validation rules listed in that file, the linter blew up with errors, so I suspect those validation rules weren't working as expected. Trying to keep this scope minimal to just feature parity with what we had before, which I why I selected this subset of rules that were working.